src/main/java/com/revature/controllers/MetricsController.java

```diff
@@ -0,0 +1,128 @@
+package com.revature.controllers;
+
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.Set;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.revature.entities.Module;
+import com.revature.services.ContentService;
+import com.revature.services.ModuleService;
+import com.revature.services.SearchService;
+import com.revature.services.TimegraphService;
+import com.revature.util.MetricsData;
+import com.revature.util.TimeGraphData;
+
+
+@CrossOrigin
+@RestController
+@RequestMapping("/metrics")
+public class MetricsController {
+	// reports mapping for metrics
+	@Autowired
+	ModuleService moduleService;
+	@Autowired
+	ContentService contentService;
+	@Autowired
+	SearchService searchService;
+	@Autowired
+	TimegraphService timegraphService;
+		
+	
+	
+	@PostMapping("/obtain/{timeFrame}")

```

Comment:
why obtain?

Reply:
Needs to be changed to metrics. I'll make the change to resolve that.